source/includes/reference/_borderless.md

```diff
@@ -442,13 +524,51 @@ X-idempotence-uuid                | Unique identifier assinged by you. Used for
 
 
 
-## Get Available Currencies
+## Get Available Currency Codes
+
+Get list of currencies that your can hold in your borderless account.
 
 > Example Request:
 
 ```shell
-curl -X GET https://api.sandbox.transferwise.tech/v1/borderless-accounts/balance-currencies \
+curl -X GET https://api.sandbox.transferwise.tech/v2/borderless-accounts-configuration/profiles/<your_profile_id>/available-currencies \
```

Comment:
I guess it looks better:
```
curl -X GET https://api.sandbox.transferwise.tech/v2/profiles/{id}/borderless-accounts/available-currencies
```
I don't understand the reason we introduce the `borderless-accounts-configurations` path. As long as we are not allowing them to create a `borderless-accounts-configuration` resource, i see no point in having this.

Reply:
It is similar to the dynamic requirements endpoints that aren't actual resources. But I see your point.